pkgs/applications/misc/bottles/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib, fetchFromGitHub
+, meson, ninja, pkg-config, wrapGAppsHook
+, desktop-file-utils, gsettings-desktop-schemas, libnotify
+, python3Packages, gettext
+, appstream-glib, gdk-pixbuf, glib, gobject-introspection, gspell, gtk3
+, steam-run-native
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "bottles";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "bottlesdevs";
+    repo = pname;
+    rev = "b0b5a259d3bb5cff89cd6dd7e7be3e6fcf3c7776";
+    sha256 = "1hbjnd06h0h47gcwb1s1b9py5nwmia1m35da6zydbl70vs75imhn";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook
+    gettext
+    appstream-glib
+    gobject-introspection
```

Comment:
Should this be in both inputs?

Reply:
`gobject-introspection`? No, that's no longer needed… :grin:

I've created the derivation initially for an older version of `bottles`, which didn't work without `gobject-introspection` in both.